tests/antares/services/local_services/test_area.py

```
@@ -223,6 +224,76 @@ def test_st_storage_and_ini_have_custom_properties_92(self, local_study_92):
+
+        # why ?
+        storage.get_pmax_injection()
+        storage.get_pmax_withdrawal()
+        storage.get_lower_rule_curve()
+        storage.get_upper_rule_curve()
+        storage.get_storage_inflows()
+
+        # v9.2
+        storage.get_cost_injection()
+        storage.get_cost_withdrawal()
+        storage.get_cost_level()
+        storage.get_cost_variation_injection()
+        storage.get_cost_variation_withdrawal()
+
+        # then
+        matrix_default = pd.DataFrame(matrix_tool.default_series)
+        assert storage.get_cost_injection().equals(matrix_default)
+        assert storage.get_cost_withdrawal().equals(matrix_default)
+        assert storage.get_cost_level().equals(matrix_default)
+        assert storage.get_cost_variation_injection().equals(matrix_default)
+        assert storage.get_cost_variation_withdrawal().equals(matrix_default)
+
+    def test_update_matrices_92(self, tmp_path: Path, local_study_92) -> None:
+        # given
+        st_storage_name = "storage_ts"
+        local_study_92.get_areas()["fr"].create_st_storage(st_storage_name)
+        # Checks all matrices exist
+        storage = local_study_92.get_areas()["fr"].get_st_storages()[st_storage_name]
+
+        # when
+        matrix = pd.DataFrame(data=8760 * [[3]])
+
+        # then
+        storage.set_cost_injection(matrix)
+        assert storage.get_cost_injection().equals(matrix)
+        storage.set_cost_withdrawal(matrix)
+        assert storage.get_cost_withdrawal().equals(matrix)
+        storage.set_cost_level(matrix)
+        assert storage.get_cost_level().equals(matrix)
+        storage.set_cost_variation_injection(matrix)
+        assert storage.get_cost_variation_injection().equals(matrix)
+        storage.set_cost_variation_withdrawal(matrix)
+        assert storage.get_cost_variation_withdrawal().equals(matrix)
+
+    def test_update_matrices_wrong_format_92(self, tmp_path: Path, local_study_92) -> None:
```

We should add 2 tests: 
1) Ensures the cost matrices are not written and cannot be `get` with a 8.8 study
2) Ensures the cost matrices cannot be modify with a 8.8 study
In such cases we should raise an explicit error msg

---

Done, all new "getter"/"setter" (v9.2) are not allowed by `get_storage_matrix `and `set_storage_matrix `for a v8.8 study

new tests added : 
- `test_creation_matrices_not_allowed_88`
- `test_update_matrices_88_not_allowed`